portfolio/src/main/webapp/script.js

```diff
@@ -31,7 +31,12 @@ const initialiseNavigation = () => {
     item.addEventListener('click', function() {
       navigate(this.getAttribute('data-link'));
     })
-  }
+  };
+  
+  const sectionSelector = document.getElementById("section-selector");
+  sectionSelector.addEventListener("change", function(e) {
```

Comment:
nit: add a space between `function` and `(`

Reply:
Does this change also apply to functions without arguements? e.g. function()

Reply:
* Updated here for commit f4a5c10 but question pending above